### DISCUSSION TITLE
GoPro Hero 10 Daily Timelapse issues

### DISCUSSION THREAD

**Discussion by logan012534** [gopro/labs#195] [General]
I have two cameras set up on an outdoor construction site. They are Hero10s running firmware 1.46 powered by external batteries. I have had intermittent issues with an identical setup on another job. The old, still existing issue is sometimes the camera will get stuck in a feedback loop- after scanning the QR code (during the hours where it would be running) it will acknowledge, go to sleep, immediately wake back up, go to sleep, wake back up, and do this perpetually. I have not found a fix, I will power down, rescan the QR, and sometimes reload the firmware until I can get it to stop.

The new issue on the new site is that after the QR scan, the cameras acknowledged and apparently never started and powered down. I returned to the site days later and the cameras were completely off. They still had battery power, but had not recorded any files. Has anyone experienced this? I have attached the screenshot of the QR command below. Thank you.

![Screen Shot 2023-01-12 at 10 25 48 AM](https://user-images.githubusercontent.com/108816213/212110548-03bb54cb-7dcc-4197-a235-bc34706cfda2.jpg)

**Comment by dnewman-gpsw** [comment]
That is a fairly reliable command, so it is best to look at the log file to see what might be going wrong.  The log file is in MISC/qrlog.txt on the sd card.

**Comment by logan012534** [comment]
Thanks for the response. As they are running now, I will check the log file when I do the scheduled card dump and advise from there.

**Comment by Pallenwht** [comment]
I am having the same problem with my 2 Hero8's. (Discussion #183) The construction TL works fine for the first interval, but after that the cameras turn back on after about 10 seconds but do not shoot an image, just a screen telling me the time till the next shot then powers down for another 10 seconds.  David is assisting me with this, but we have yet to find a resolve.

**Comment by logan012534** [comment]
Hi, I am attaching the log files from several cameras below. Now experiencing the same issue from the original post on a second jobsite where the cameras have been running since June. Four camera setup and they have mostly been reliable, every now and then one specific camera would get stuck in a feedback loop and every now and then some cameras will have stopped recording for unknown reason. I recently went down to swap external batteries, and three of the three cameras I checked had all stopped recording. One of the cameras ran for two days before quitting, and the other two never started recording after the QR scan. I visit the site biweekly so there was a lot of missed content. I opted to factory reset all cameras and run the internal timelapse modes on all projects for now. I think there is a lot of potential to these GoPro labs functions but I have had too many issues and missed footage to continue running the risk. Happy to provide any additional info that I can to troubleshoot.

[qrlog_1.txt](https://github.com/gopro/labs/files/10547340/qrlog_1.txt)
[qrlog_4.txt](https://github.com/gopro/labs/files/10547341/qrlog_4.txt)
[qrlog_3.txt](https://github.com/gopro/labs/files/10547342/qrlog_3.txt)
[qrlog_2.txt](https://github.com/gopro/labs/files/10547343/qrlog_2.txt)

**Comment by dnewman-gpsw** [comment]
These are clear examples on the internal timer failing for reasons unknown.  Falling back to the internal timelapse mechanisms will be the mostly reliable but without the daily upload.  The best compromise is proposed here https://github.com/gopro/labs/discussions/188   If you can use a daily external on/off timers, then can the camera wake on power, to run any script you need as a fresh camera every day, super reliable.

**Reply by Pallenwht** [reply]
RE: Discussion #183  You sent me a "Reset QR code" and I ran that. I have not had any problems with the TL code since then, the construction TL works fine. EXCEPT:  I went to swap memory cards yesterday (2/13) and found that the program quit running on 2/6 so I lost almost 2 weeks of shots. Luckily it has been a slow period for the construction. Attached is the qrlog for 2/6 when it shutdown.  
[qrlog.txt](https://github.com/gopro/labs/files/10737343/qrlog.txt)

qrlog copy.txt

**Reply by dnewman-gpsw** [reply]
This log doesn't show the use an external timer, just internal wake events.

**Comment by logan012534** [comment]
Thanks for the quick reply. Is there any specific timer that is recommended? I have not deployed those in a setup before.

**Reply by dnewman-gpsw** [reply]
For A/C powered near a WiFi network, any of the "wifi smart plugs" work well.  Set up with your smartphone.  USB powered timers I'm about to try https://www.amazon.com/Micro-USB-Smart-Adaptor-Compatible/dp/B083BKBKW5

**Comment by logan012534** [comment]
That's slick. Unfortunately we had to abandon the daily uploads because the wifi we installed was not reliable, so that would rule out a smart device. I will look around, thanks.

**Comment by Pallenwht** [comment]
Has there been any resolve to the issue of the construction timelapse quiting after a week or two?  I am experiencing  this problem and have lost some rather important images of this ongoing timelapse job. I guess I will need to switch to internal timelapse on my 2 Hero8's. For me battery power is not s real concern if I have to allow uneedind recordings  at night. (solar recharge on a 3.3ah battery).
Please let me know.   Paul

**Reply by dnewman-gpsw** [reply]
Nothing has changed on HERO8, so definitely use internal timelapse modes if you have the power to support that. Also using scripted internal timelapse works very well, so only timing lapse during the day is straight forward using this example without upload https://gopro.github.io/labs/control/dailytl/ (it works for HERO8)

**Reply by logan012534** [reply]
I tried the smart plug option, but during testing it would not sustain the schedule outside of the network that it was provisioned on. I don't have internet access on the site so was testing to see if it needs to check in or phone home or something of that sort to keep the start/stop schedule. I opted to run the internal timelapse settings on the Hero10 but was having issues on those as well, with them randomly powering down and stopping recording. GoPro technical support wasnt much help, besides saying the cameras arent designed to run for days on end. I dont have problems with every camera deployed but the issues arent predictable either so it is hard to diagnose. I think there is a lot of potential with the GoPro labs, but I've lost a lot of content due to the bugs. I ended up swapping them out for DJI Osmo Action 3s and have not had any issues yet. They do not offer the same variety of duration options between frames for the timelapse, but it's better than not capturing any content at all.

**Comment by Pallenwht** [comment]
I wonder if temperature or weather conditions affect the camera operation.
I could think that temperature extremes could affect camera operation, but
not much can be done about that. My cameras are in Pelco enclosures,
weather proof BUT no temp control.  "Cameras are not designed to run for
days on end" meaning that maybe Gopro tested their cameras for that
extreme. Guess we're just stuck with checking the cameras every 4 days (my
last failure duration)
- Not easy!

**Reply by dnewman-gpsw** [reply]
Report the logs, as we can improve with feedback.
